### PR TITLE
Add new cards from Resistance conversion kit preview

### PR DESCRIPTION
**Pull request by FreakyDug**
https://www.fantasyflightgames.com/en/news/2018/10/3/continuing-the-fight/

### CODE REVIEW

File: data/pilots/resistance/scavenged-yt-1300.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{
+  "name": "Scavanged YT-1300",
```
Comment (guidokessels) [inline data/pilots/resistance/scavenged-yt-1300.json]: This should be `Scavenged`

File: data/upgrades/crew.json [hunk 2]
```diff
@@ -1061,5 +1061,85 @@
     ],
     "cost": { "value": 3 },
     "restrictions": [{ "factions": ["Scum and Villainy"] }]
+  },
+  {
+    "name": "C-3PO",
+    "limited": 1,
+    "xws": "c3po-crew-swz19",
```
Comment (guidokessels) [inline data/upgrades/crew.json]: I think this should be `c3po-crew`.

File: data/pilots/resistance/scavenged-yt-1300.json (hunk 1 above)
Comment (FreakyDug) [reply data/pilots/resistance/scavenged-yt-1300.json]: Woops.

File: data/upgrades/crew.json [hunk 3]
```diff
@@ -1061,5 +1061,85 @@
...
+          }
+        ]
+      }
+    ],
+    "restrictions": [{ "factions": ["Resistance"] }]
+  },
+  {
+    "name": "Han Solo",
+    "limited": 1,
+    "xws": "hansolo-crew-swz19",
```
Comment (guidokessels) [inline data/upgrades/crew.json]: This should be `hansolo-crew`

File: data/upgrades/crew.json (hunk 2 above)
Comment (FreakyDug) [reply data/upgrades/crew.json]: Oh, good point.

**Comment by guidokessels** [comment]
Thanks a lot! 🎉

**Pull request merged**